documentation/en/api-v1-unstable-methods.md

```diff
@@ -2064,6 +2065,50 @@ Response:
 ]
 ```
 
+### EthTraceFilter
```

Comment:
@snissn the examples here could be improved I think if you add some realistic entries to api/docgen/docgen.go; they aren't terrible as they are but they're pulling together smaller scalar examples currently.

Reply:
thanks! cool! here we are 1323ec762